src/TwigJs/JsCompiler.php

```diff
@@ -255,12 +256,13 @@ public function __construct(Environment $env)
         $this->testCompilers = array(
             'defined' => new DefinedCompiler(),
             'divisibleby' => new DivisibleByCompiler(),
-            'empty' => new EmptyCompiler(),
-            'even' => new EvenCompiler(),
-            'none' => new NoneCompiler(),
-            'null' => new NullCompiler(),
-            'odd' => new OddCompiler(),
-            'sameas' => new SameAsCompiler(),
+            'empty'       => new EmptyCompiler(),
+            'even'        => new EvenCompiler(),
+            'none'        => new NoneCompiler(),
+            'null'        => new NullCompiler(),
+            'odd'         => new OddCompiler(),
+            'sameas'      => new SameAsCompiler(),
```

Comment:
`sameas` should be removed. as it's deprecated in 1.x and removed in 2.x

Reply:
Removed